models/repo_list_test.go

```diff
@@ -60,10 +55,133 @@ func TestSearchRepositoryByName(t *testing.T) {
 		Page:     1,
 		PageSize: 10,
 		Private:  true,
-		Searcher: &User{ID: 14},
 	})
 
-	assert.NotNil(t, repos)
 	assert.NoError(t, err)
 	assert.Equal(t, int64(3), count)
+	assert.Len(t, repos, 3)
+
+	// Get all public repositories by name
+	repos, count, err = SearchRepositoryByName(&SearchRepoOptions{
```

Comment:
This is a lot of duplicated code, maybe we should make this a [table-driven test](https://github.com/golang/go/wiki/TableDrivenTests).

Reply:
Same as above ;-)